src/forms/input/Input.tsx

```diff
@@ -66,6 +70,8 @@ export class Input extends React.PureComponent<InputProps> {
                 <input
                     type={type || "text"}
                     id={name}
+                    value={value}
+                    defaultValue={defaultValue}
```

Comment:
As per my knowledge you can not set 'defaultValue' directly to input HTML tag. 

You can set 'defaultValue' programmatically once, and then let user do whatever.With uncontrolled values, the value attribute (value={value here}) inside the input element will override any values that are typed in so in order to work around that, React/JavaScript provides another attribute called defaultValue (defaultValue={value here}) that will pre-populate the input field with the defaultValue without overriding any value input by the user.

Reply:
`defaultValue` prop is supported by React, you can read more about it https://reactjs.org/docs/uncontrolled-components.html.
In this case, it might not be required though.

Reply:
@kulakshay  Thanks for clarifying this.